bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Triboson_WkkWr/Triboson_M3000_R0.5/Triboson_M3000_R0.5_run_card.dat

```diff
@@ -0,0 +1,270 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#*********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  #False     = gridpack  !True = setting up the grid pack
```

Comment:
don't enable settings. there will always be a default. better to know. anyway, the setting will be centrally handled. i will change the line.

Reply:
Ah actually, the template file is this one: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Triboson_WkkWr/Triboson_M_R/Triboson_M_R_run_card.dat